lxd/container_lxc.go

```diff
@@ -1099,6 +1101,45 @@ func (c *containerLXC) startCommon() (string, error) {
 			if err != nil {
 				return "", fmt.Errorf("Failed to add cgroup rule for device")
 			}
+		} else if m["type"] == "usb" {
+			if usbs == nil {
+				usbs, err = deviceLoadUsb()
+				if err != nil {
+					return "", err
+				}
+			}
+
+			for _, usb := range usbs {
+				if usb.vendor != m["vendorid"] || (m["productid"] != "" && usb.product != m["productid"]) {
+					continue
+				}
+
+				err = lxcSetConfigItem(c.c, "lxc.cgroup.devices.allow", fmt.Sprintf("c %d:%d rwm", usb.major, usb.minor))
+				if err != nil {
+					return "", err
+				}
+
+				m["major"] = fmt.Sprintf("%d", usb.major)
+				m["minor"] = fmt.Sprintf("%d", usb.minor)
+				m["path"] = usb.path
+
+				/* it's ok to fail, the device might be hot plugged later */
+				_, err := c.createUnixDevice(m)
+				if err != nil {
+					shared.Log.Warn("failed to create usb device", log.Ctx{"err": err, "device": k})
```

Comment:
Should be downgraded to Debug since having the device hot-plugged later is supported.